test/test_ugrid.py

```diff
@@ -82,8 +82,14 @@ def test_standardized_dtype_and_fill(self):
         ux_grid3 = ux.Grid(xr_grid3)
 
         # check for correct dtype and fill value
-        grids = [ux_grid1, ux_grid2, ux_grid3]
-        for grid in grids:
+        grids_with_fill = [ux_grid2]
+        for grid in grids_with_fill:
+            assert grid.Mesh2_face_nodes.dtype == INT_DTYPE
+            assert grid.Mesh2_face_nodes._FillValue == INT_FILL_VALUE
+            assert INT_FILL_VALUE in grid.Mesh2_face_nodes.values
+
+        grids_without_fill = [ux_grid1, ux_grid2]
+        for grid in grids_without_fill:
```

Comment:
Why `ux_grid2` is showing in both `grids_without_fill` and `grids_with_fill` and where is `ux_grid3`?

Reply:
Good catch